client/my-sites/checkout/composite-checkout/test/we-chat-processor.ts

```diff
@@ -54,6 +54,7 @@ describe( 'weChatProcessor', () => {
 			create_new_blog: true,
 			currency: 'USD',
 			extra: [],
+			is_jetpack_checkout: false,
```

Comment:
Thanks for modifying all of these tests. I should really extract some of these expectations into a helper. 🤔